Fix Sentence Construction (#23060)
github/docs: content/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork.md

  - Pull requests
shortTitle: Allow changes to a branch
---

> When a user creates a pull request from its fork, the user generally has the authority to decide who can commit to the pull request's compare branch. If the pull request authors want greater collaboration, they can grant upstream repository maintainers (i.e., anyone with push access to the upstream repository) permission to commit to the pull request's compare branch. To learn more about upstream repositories, see "[About forks](/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks)."

Reviewer: ```suggestion
When a user creates a pull request from a fork that they own, the user generally has the authority to decide if other users can commit to the pull request's compare branch. If the pull request author wants greater collaboration, they can grant maintainers of the upstream repository (that is, anyone with push access to the upstream repository) permission to commit to the pull request's compare branch. To learn more about upstream repositories, see "[About forks](/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks)."
```
